.circleci/config.yml

```diff
@@ -14,7 +14,7 @@ parameters:
     # 3. Commit this change to the PR branch where the changes exist.
     current_golden_images_hash:
         type: string
-        default: 7ecf35bddd42938e48adcb647e029bb9c75f8663
+        default: 86d88ee8d5440f8b75b02b63981fb3e5bc045df3
```

Comment:
Screenshots for 2 new stories. Plus 11 screenshots (action menu, picker, submenu),  have been updated because we should no long force focus by setting the open property. Only a user action should trigger a menu focus.